### PR TITLE
Add customization  docs for Leptonx  lite

### LINKED ISSUE #444: Table Of Contents of current document is not navigating on page scroll

### PR DESCRIPTION
**Pull request by mahmut-gundogdu**
resolves #444

### CODE REVIEW

File: docs/en/Themes/LeptonXLite/Angular.md [hunk 1]
```diff
@@ -82,3 +82,165 @@ To change the logos and brand color of `LeptonX`, simply add the following CSS t
...
+
+## Customization
+
+
+### Layouts
+
+Angular version of LeptonX Lite provides **layout components** for your **user interface** on [ABP Framework Theming](https://docs.abp.io/en/abp/latest/UI/Angular/Theming). You can use layouts to **organize your user interface**. You can replace the **layout components** and some parts of the **layout components** with the [ABP replaceable component  system](https://docs.abp.io/en/abp/latest/UI/Angular/Component-Replacement).
+
+
+The main responsibility of a theme iss to **provides** the layouts. There are **three pre-defined layouts that must be implemented by all the themes:**
```
Comment (ismcagdas) [inline docs/en/Themes/LeptonXLite/Angular.md]:
```suggestion
The main responsibility of a theme is to **provide** the layouts. There are **three pre-defined layouts that must be implemented by all the themes:**
```

**Review by ismcagdas — CHANGES_REQUESTED** [review]
We can also remove "Page Alerts Component" section if that part is not replacable.

**Comment by ismcagdas** [comment]
@braim23 could you also check this PR when you have time.

**Comment by braim23** [comment]
> @braim23 could you also check this PR when you have time.

I'll do that once I'm done with the issue I'm working on

**Review by braim23 — APPROVED** [review]

**Review by ismcagdas — APPROVED** [review]

**Pull request merged**